libs/core/src/repo/schema_v3.rs

```diff
@@ -0,0 +1,123 @@
+use crate::CoreError;
+use hmdb::transaction::Transaction;
+use lockbook_shared::account::Account;
+use lockbook_shared::file_like::FileLike;
+use lockbook_shared::file_metadata::Owner;
+use lockbook_shared::signed_file::SignedFile;
+use lockbook_shared::{document_repo, SharedError};
+use serde::{Deserialize, Serialize};
+use std::fs;
+use uuid::Uuid;
+
+use super::schema_v2::CoreV2;
+
+pub type Tx<'a> = transaction::CoreV3<'a>;
+
+#[derive(Debug, Clone, Hash, PartialEq, Eq, Serialize, Deserialize)]
+pub struct OneKey;
+
+hmdb::schema! {
+    CoreV3 {
+        account: <OneKey, Account>,
+        last_synced: <OneKey, i64>,
+        root: <OneKey, Uuid>,
+        local_metadata: <Uuid, SignedFile>,
+        base_metadata: <Uuid, SignedFile>,
+        read_activity: <Uuid, usize>,
+        write_activity: <Uuid, usize>,
```

Comment:
thinking about whether this should be 1 or 2 tables, also maybe pop these guys at the bottom

Reply:
I think 2 tables is fine